noxfiles/docker_nox.py

```diff
@@ -88,20 +90,21 @@ def build(session: nox.Session, image: str, machine_type: str = "") -> None:
             tag_name = get_current_tag()
         if image == "test":
             tag_name = "local"
-        privacy_center_image_tag = f"{PRIVACY_CENTER_IMAGE}:{tag_name}"
-        sample_app_image_tag = f"{SAMPLE_APP_IMAGE}:{tag_name}"
-
         session.log("Building extra images:")
+        # prod/test PC is pushed to ethyca/fides-privacy-center image
```

Comment:
This is already documented in `push`, I don't think it is useful documentation to have in `dev`

Reply:
good point, I'll remove